node/forest_libp2p/src/chain_exchange/provider.rs

```diff
@@ -169,8 +174,19 @@ mod tests {
     async fn compact_messages_test() {
         let (cids, db) = populate_db().await;
 
+        let gen_block = BlockHeader::builder()
+            .epoch(1)
+            .weight(2_u32.into())
+            .messages(Cid::new_v1(DAG_CBOR, Identity.digest(&[])))
+            .message_receipts(Cid::new_v1(DAG_CBOR, Identity.digest(&[])))
+            .state_root(Cid::new_v1(DAG_CBOR, Identity.digest(&[])))
+            .miner_address(Address::new_id(0))
+            .build()
+            .unwrap();
+        let genesis_ts = Tipset::new(vec![gen_block]).unwrap();
```

Comment:
Like in your other test, this is not a great genesis header. Try using a simpler one or use the one from calibnet.

Reply:
Simplified.